thoth-export-server/Cargo.toml

```diff
@@ -20,6 +20,7 @@ env_logger = "0.7.1"
 lazy_static = "1.4.0"
 log = "0.4.14"
 paperclip = { version = "0.7.0", features = ["actix-base", "actix4", "uuid", "v2"] }
+regex = "1.6.0"
```

Comment:
This one only seems to be used in tests, so it'd be good to add it as dev dependency instead